config/thoth.yaml

```diff
@@ -44,3 +44,12 @@ repositories:
     slug: thoth-station/jupyter-notebook
   - <<: *update_config
     slug: thoth-station/sesheta
+  - slug: user-cont/conu
+    token: '{KEBECHET_TOKEN}'
+    managers:
+      - name: update
+        configuration:
+          labels: 
```

Comment:
I tried to look up what does this configuration mean, could you please elaborate? Will the bot set these labels on a PR?

Reply:
ja

Reply:
@TomasTomecek if you want to have different labels, just let us know

Reply:
do we need to create them first?

Reply:
> do we need to create them first?

I think they are created by github automatically if you request to use them on github API.

Reply:
cool, let us know if github complains